test/unit/views/MovementConfirmationViewSpec.scala

```diff
@@ -93,6 +71,9 @@ class MovementConfirmationViewSpec extends UnitViewSpec with CommonMessages {
 
   "Departure Confirmation Page" should {
 
+    implicit val request = journeyRequest(DepartureAnswers())
+    val departureConfirmationView = movementConfirmationPage(Choice.Departure, consignmentReferences)
+
     "have title" in {
 
       departureConfirmationView.getElementsByTag("title").first().text() must include(messages("title.format"))
```

Comment:
Could you update this line as the line 39 ?

Reply:
Yes.  Ignore this file for now.  It's all wrong (well, nearly all)

Reply:
Spec now all fixed